lib/output_helper.rb

```diff
@@ -314,17 +314,19 @@ def self.generate_geometry(solution)
       expected_geometry.map!(&:to_sym)
       solution[:result].collect{ |result|
         geojson = {}
-
-        geojson[:partitions] = generate_partitions_geometry(result) if expected_geometry.include?(:partitions)
-        geojson[:points] = generate_points_geometry(result)
-        geojson[:polylines] = generate_polylines_geometry(result) if expected_geometry.include?(:polylines) && ENV['OPTIM_GENERATE_GEOJSON_POLYLINES']
+        vehicle_color_indices = result[:routes]&.map&.with_index{ |route, index|
+          [route[:original_vehicle_id], index]
+        }&.to_h
```

Comment:
May be a small comment about why we do this -- like  "if there is vehicle partition, the geojson object colors will be based on vehicle indices"